source/docs/articles/drupal/adding-yml-file.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Adding a services.yml File
+description: Learn how to add a services.yml file for Drupal 8.
+category:
+  - developing
+  - drupal
+keywords: drupal, services.yml, yml file
+---
+Drupal 8 includes the `sites/default/default.services.yml` configuration file to define services used, eliminating the need to hack core to swap functionality. To create and/or modify this file, you'll need to tweak some permissions, otherwise you will receive a **Permission denied** error.
+
+<div class="alert alert-danger">
+<h4>Warning</h4>
+It is not possible to modify these permissions with Git. Users working via Git should finish up their code changes and briefly switch to <a href="/docs/articles/sites/code/developing-directly-with-sftp-mode">SFTP Mode</a> to change the permissions.
+</div>
+
+## Create `services.yml`
+Using your preferred SFTP client, copy `sites/default/default.services.yml` into a new file named `sites/default/services.yml`. Do not simply rename the original file. The install script requires both files to be present at the time of installation.
+
+## Change Permissions for `sites/default`
+Depending on your SFTP client, use 'get info' or 'file attribute' to change permissions of the `sites/default` directory to 777 (read + write + execute). Writable files, such as `services.yml`, should be changed to 666 (read + write).
+
+You can now upload the file and run the installation script as normal. These permissions will automatically be set back to default after the script has finished. If you are modifying this file post installation, you will need to set the permissions back to the default (555 for directories, 444 for files) once you have made the desired changes.
```

Comment:
'after the script has finished' should be 'after the installation script has finished'; otherwise, the user might wonder if you meant the "upload file" script or something else.  There is no such thing, of course, but if the user didn't know that, it could be confusing.